frontend/packages/vkt/src/components/publicEnrollment/steps/PreviousEnrollment.tsx

```diff
@@ -2,108 +2,190 @@ import {
   Collapse,
   FormControl,
   FormControlLabel,
+  FormHelperText,
   FormLabel,
   Radio,
   RadioGroup,
 } from '@mui/material';
-import { ChangeEvent, useState } from 'react';
+import { ChangeEvent, useEffect, useState } from 'react';
 import { CustomTextField } from 'shared/components';
 import { TextFieldTypes } from 'shared/enums';
-import { InputFieldUtils } from 'shared/utils';
+import { TextField } from 'shared/interfaces';
+import { getErrors, hasErrors } from 'shared/utils';
 
 import { useCommonTranslation, usePublicTranslation } from 'configs/i18n';
 import { useAppDispatch } from 'configs/redux';
 import { PublicEnrollment } from 'interfaces/publicEnrollment';
 import { updatePublicEnrollment } from 'redux/reducers/publicEnrollment';
 
+interface PreviousEnrollmentField {
+  previousEnrollment?: string;
+}
+
+enum HasPrevious {
```

Comment:
Ehkä turhan viilausta mutta jos joku projektin ulkopuolinen kaveri lukisi tätä niin `HasPrevious` on paljon monitulkinnaisempi kuin esim. `PreviouslyEnrolled`. is/has voi olla hyvä pitää kuvastamassa asioita, jotka on pelkästään booleaneita.